server/modules/MainServer.js

```diff
@@ -23,6 +24,17 @@ let serverInstance;
 
 const APP_VERSION = require('config').get('APP_VERSION');
 
+function usernameMiddleware(req, res, next) {
+  try {
```

Comment:
Could we move `usernameMiddleware` (and perhaps some associated logging/winston stuff?) into something like a `loggingMiddleware` module to avoid bloating the main server module?